hack/disconnected.sh

```
@@ -0,0 +1,18 @@
+#!/bin/bash
+DESTINATION_PATH=$1
```

Can we have this fail if not specified?  E.g. `...=${1?"error message "}`

---

I dont see why not. Right now its an optional var but we can force require it

---

Ah - I missed it being optional - nvm then.